### PR TITLE
add SWUPDATE build in CI

### PR DESCRIPTION
**Pull request by WAvdBeek**

### CODE REVIEW

**Comment by WAvdBeek** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by WAvdBeek** [comment]
fixes issue:
https://github.com/iotivity/iotivity-lite/issues/3

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -142,3 +142,20 @@ jobs:
...
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE,  so your job can access it
+      - uses: actions/checkout@v2
+      # Runs a set of commands using the runners shell
+      - name: make secured with software update (using boost)
+        run: |
+          # the setup
+          #sudo apt-get -y install libboost-all-dev
```
Comment (tiniuclx) [inline .github/workflows/build.yml:158]: This is supposed to be commented out, right?

**Review by tiniuclx — APPROVED** [review]
Looks good to me, as long as the line I pointed out is commented out on purpose!

File: .github/workflows/build.yml (hunk 1 above)
Comment (WAvdBeek) [reply .github/workflows/build.yml:158]: yes, I have removed boost, now using regex to do the url validation, avoiding the linking with boost

**Pull request merged**